tests/requirements.in

```diff
@@ -0,0 +1,5 @@
+pytest==3.4.0
```

Comment:
That's not how it works. In the `.in` file, you specify lower and (if needed) upper bounds, but no exact versions. Given this, `pip-compile` can generate a `.txt` which contains only exact versions. This way, without changing the `.in` file, the version bounds can be updated to fixed versions by simply re-running `pip-compile`.

Reply:
I dont have hard deps now would it work defaulting to the same version as a lower limit ?